plugins/arElasticSearchPlugin/lib/model/arElasticSearchInformationObject.class.php

```diff
@@ -60,6 +60,8 @@ public function populate()
 
   public function recursivelyAddInformationObjects($parentId, $totalRows, $options = array())
   {
+    $skip = isset($options['skip']) ? $options['skip'] : null;
```

Comment:
If you set `$skip` to zero by default, it makes the conditional check later easier.

Reply:
Good call.